TestAssets/TestProjects/SimpleCache/SimpleCache.csproj

```diff
@@ -0,0 +1,7 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+    <ItemGroup>
+        <PackageReference Include="System.Private.Uri" Version="4.4.0-beta-24821-02" />
+        <PackageReference Include="Microsoft.NETCore.CoreDisTools" Version="1.0.1-prerelease-00001" />
```

Comment:
Does this package need to be referenced in order to run the `ComposeCache` target?  Should there be a test of a "normal" test app, like [https://github.com/dotnet/sdk/blob/master/TestAssets/TestProjects/SimpleDependencies/SimpleDependencies.csproj](https://github.com/dotnet/sdk/blob/master/TestAssets/TestProjects/SimpleDependencies/SimpleDependencies.csproj)?